org.eclipse.jdt.debug.tests/tests/org/eclipse/jdt/debug/tests/core/LineTrackerTests.java

```diff
@@ -181,6 +181,14 @@ public void testHyperLink() throws Exception {
 	    try {
 			ConsoleLineTracker.setDelegate(this);
 			fTarget = launchAndTerminate("ThrowsNPE");
+			long startTime = System.nanoTime();
+			long timeOut = 6000;
+			while(ConsoleLineTracker.getDocument()==null) {
+				if (System.currentTimeMillis() - startTime > timeOut) {
```

Comment:
compare to System.nanoTime(); and multiply timeOut by 1_000_000

Reply:
Okay..👍